he.json

```diff
@@ -0,0 +1,297 @@
+                    "twitter-line1" : "עברנו להשתמש בדיסקורד לצורך התחברות. אתה כרגע משתמש בחשבון מדור קודם שעלול להיפסק בעתיד ושכבר עכשיו בעל פונקציונליות מוגבלת.",
+                    "twitter-line2" : "כדי לעבור לחשבון דיסקורד, לחץ על הכפתור מטה"
+                },
+                "ratings" : {
+                    "title" : "דירוגים",
+                    "none" : "אין לך דירוגים."
+                },
+                "destroy" : {
+                    "title" : "השמד חשבון",
+                    "line1" : "אם אינך רוצה עוד את חשבונך, תוכל למחוק אותו ואת כל הנתונים שלו [כאן](/account/destroy)"
+                }
+            },
+            "oauth" : {
+                "title" : "התחבר לfnbr.co",
+                "discord-button" : "התחבר בעזרת דיסקורד",
+                "legacy-text" : "לחלופין, אם יש לך חשבון דור קודם משנת 2018 הזקוק להעברה:",
+                "twitter-button" : "התחבר בעזרת טוויטר",
+                "privacy-text" : "על ידי כניסה אתה מסכים לנו לאחסן נתונים המשויכים לחשבונך כמפורט [במדיניות הפרטיות](/privacy) שלנו."
+            },
+            "username" : {
+                "title" : "ברוך הבא לfnbr.co",
+                "line1" : "אתה מחובר בעזרת דיסקורד. כדי ליצור אינטרקציה עם אחרים באתר, תצטרך לבחור שם משתמש",
+                "nameTaken" : "שם המשתמש שלך [attemptedName] נלקח על ידי משתמש אחר, ולכן הוספנו לו את מאבחן הדיסקורד שלך ואנו ממליצים לך להשתמש בשם זה.",
+                "randomName" : "שם המשתמש שלך [attemptedName] נלקח על ידי משתמש אחר, ולא הצלחנו ליצור לך שם משתמש יחודי בעזרת מאבחן משתמש הדיסקורד שלך.",
+                "form-username" : "שם משתמש",
+                "form-criteria" : "אנא בחר שם משתמש בין 2 ל-32 תווים, אשר יכיל אותיות באנגלית,  קווים תחתונים ומקפים בלבד.",
+                "button-text" : "קבע את שם המשתמש שלי"
+            },
+            "locked" : {
+                "title" : "ההתחברות נכשלה",
+                "line1" : "אינך יכול להתחבר לfnbr.co מכיוון [שחשבונך ננעל](class=text-danger)"
+            },
+            "migrated" : {
+                "title" : "חשבונך הועבר",
+                "line1" : "התחברת בעזרת טוויטר בתור ",
+                "line2" : "אך משתמש זה הועבר להתחברות בעזרת דיסקורד."
+            },
+            "notFound" : {
+                "title" : "לא נמצא חשבון",
+                "line1" : "התחברת בעזרת טוויטר בתור ",
+                "line2" : "אך משתמש זה לא התחבר לפני כן וכרגע אנו תומכים רק בכניסה בעזרת טוויטר רק לשם העברת החשבון.",
+                "line3" : "לחלופין, תוכל [להתחבר באמצעות דיסקורד](/oauth/discord)"
+            }
+        },
+        "errors" : {
+            "404" : {
+                "title" : "שגיאה",
+                "line1" : "המשאב שביקשת אינו קיים, או שאין לך הרשאה להציג אותו."
+            },
+            "500" : {
+                "title" : "אוי לא!",
+                "line1" : "משהו השתבש במהלך עיבוד בקשתך, אנא נסה שוב מאוחר יותר!",
+                "line2" : "תוכל לנסות ולהתחבר שוב [כאן](/oauth)"
```

Comment:
It's not allowed to separate the link name (within the square brackets `[...]`) from the link url (inside the round brackets `(...)`). It must be in this format: `[link name](link url)`